docs/cep/tutorials/create-stream-app.md

```diff
@@ -209,12 +211,17 @@ Query Parameters:
 ### C8DB
```

Comment:
Avoid stacked headings, need an intro paragraph after the Table heading above. Also, I don't think we're using C8DB anymore, so this seems like the wrong heading text for this section.